### ISSUE TITLE
pam_pbssimpleauth selinux module

### ISSUE THREAD

**Issue by necrolyte2** [VDBWRAIR/vdbwrair-torque#2]

**Comment by necrolyte2** [comment]
Dumping the module here for later use

```
module pam_pbssimpleauth 1.0;

require {
        type sshd_t;
        type var_spool_t;
        class dir read;
        class file open;
}

#============= sshd_t ==============

allow sshd_t var_spool_t:dir read;
allow sshd_t var_spool_t:file open;
```

**Comment by necrolyte2** [comment]
http://docs.adaptivecomputing.com/torque/archive/3-0-2/3.4hostsecurity.php

**Comment by necrolyte2** [comment]
also need to configure access.conf

**Comment by necrolyte2** [comment]
Closed in  83df509
